webapp/src/Service/ImportExportService.php

```diff
@@ -537,12 +539,46 @@ public function importTsv(string $type, UploadedFile $file, string &$message = n
         }
     }
 
+    /**
+     * Import a JSON file
+     * @param string       $type
+     * @param UploadedFile $file
+     * @param string|null  $message
+     * @return int
+     * @throws Exception
+     */
+    public function importJson(string $type, UploadedFile $file, string &$message = null): int
+    {
+        $content = file_get_contents($file->getRealPath());
+        $data = json_decode($content, true);
+        if (json_last_error() !== JSON_ERROR_NONE) {
+            $message = 'File contents is not JSON';
```

Comment:
Maybe be a bit more verbose: first, say `is not *valid* JSON` and then append the output of `json_last_error_msg()` so that the user has some chance to understand what's wrong.